src/components/TokenSelect/index.tsx

```diff
@@ -27,6 +29,15 @@ const SearchInputContainer = styled(Row)`
   ${inputCss}
 `
 
+const TokenSelectContainer = styled.div`
+  border-radius: ${({ theme }) => theme.borderRadius.medium}em;
+  overflow: hidden;
+  padding: 0.5em 0 0;
+  @supports (overflow: clip) {
+    overflow: 'clip';
```

Comment:
is there a downside to just putting this without the @supports? just curious

Reply:
yes, it would override the "hidden" value